src/Coordinate/Ellipsoid.php

```diff
@@ -199,7 +199,7 @@ class Ellipsoid
         ),
         self::WGS84 => array(
             'name' => 'WGS 84',
-            'a'    => 6378136.0,
+            'a'    => 6378137.0,
```

Comment:
I guess the earth is a little bit bigger now

Reply:
Nope, it's always 6,378,137 [according to wikipedia](https://en.wikipedia.org/wiki/Earth_ellipsoid) (see WGS-84)